Update libaom to M132

youennf opened the pull request:
#### 4e5ce282ceb7558a4e7edd81cd510293c4740fc1
<pre>
Update libaom to M132
<a href="https://bugs.webkit.org/show_bug.cgi?id=283965">https://bugs.webkit.org/show_bug.cgi?id=283965</a>
<a href="https://rdar.apple.com/140846548">rdar://140846548</a>

Reviewed by Jean-Yves Avenard.

We enable CONFIG_AV1_DECODER as this is used by libwebrtc and libaom is more tightly removing decoder only routines.

* Source/ThirdParty/libwebrtc/Configurations/BaseTarget-libaom.xcconfig:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/config/linux/arm64-cpu-detect/config/aom_dsp_rtcd.h:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/config/linux/generic/config/aom_config.h:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/libaom: Resynced.
* Source/ThirdParty/libwebrtc/libwebrtc.xcodeproj/project.pbxproj:

Canonical link: <a href="https://commits.webkit.org/287340@main">https://commits.webkit.org/287340@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f06b82ffa577064e4b2ee04d7e527660078ebd7d

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/79048 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/58084 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/32425 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/83688 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/30263 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/81181 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/67193 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/6354 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/61871 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/19789 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/82115 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/51912 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/72053 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/42174 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/49263 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/26149 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/28627 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/70378 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/26566 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/85074 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/6373 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/4405 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/70109 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/6537 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/67925 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/69358 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/17320 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/13404 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/12181 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/6324 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/6285 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/9736 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/8077 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

jyavenard reviewed (approved):
Commit log needs more clarity on what "libaom is more tightly removing decoder only routines" means

youennf committed:
Update libaom to M132
https://bugs.webkit.org/show_bug.cgi?id=283965
rdar://140846548

Reviewed by Jean-Yves Avenard.

We enable CONFIG_AV1_DECODER as this is used by libwebrtc and libaom is more tightly removing decoder only routines.

* Source/ThirdParty/libwebrtc/Configurations/BaseTarget-libaom.xcconfig:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/config/linux/arm64-cpu-detect/config/aom_dsp_rtcd.h:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/config/linux/generic/config/aom_config.h:
* Source/ThirdParty/libwebrtc/Source/third_party/libaom/source/libaom: Resynced.
* Source/ThirdParty/libwebrtc/libwebrtc.xcodeproj/project.pbxproj:

Canonical link: https://commits.webkit.org/287340@main
